docs/_pages/data.md

```diff
@@ -82,6 +82,26 @@ When checking the equivalency of two `DataRow` objects, by default the `RowState
 
 In addition, if both the subject and the expectation are in the `DataRowState.Modified` state, then the `DataRowVersion.Original` values are also compared, separately from the `DataRowVersion.Current` values. This can be disabled using the `.ExcludingOriginalData()` equivalency assertion option.
 
+## Collections
+
+Each `DataSet` has a `DataTableCollection` called `Tables`, and each `DataTable` has a `DataColumnCollection` called `Columns` and a `DataRowCollection` called `Rows`. Some assertions can be performed on these collection types.
+
+The following assertions are in common to all three collection types:
+
+* `.Should().BeEmpty()`: Succeeds if the collection contains no items (tables, columns, rows).
+* `.Should().NotBeEmpty()`: Succeeds if the collection contains at least one item (table, column, row).
+* `.Should().ContainEquivalentOf(x)`: Succeeds if the collection contains an item (table, column, row) that is equivalent to the supplied item.
+* `.Should().NotContainEquivalentOf(x)`: Succeeds if the item does not contain any item (table, column, row) that is equivalent to the supplied item.
+* `.Should().HaveSameCount(x)`: Succeeds if the collection contains the same number of items as the supplied collection of the same type.
+* `.Should().NotHaveSameCount(x)`: Succeeds if the collection does not contain the same number of items as the supplied collection of the same type.
+* `.Should().HaveCount(x)`: Succeeds if the collection contains exactly the specified number of items.
+* `.Should().HaveCount(predicate)`: Succeeds if the predicate returns true for the number of items in the collection.
+* `.Should().NotHaveCount(x)`: Succeeds if the collection contains a different number of items than the supplied count.
+* `.Should().HaveCountGreaterThan(x)`: Succeeds if the collection contains more items than the supplied count.
```

Comment:
🔧 This seems to be out-of-date with the current contents of the PR, right?

Reply:
How so? All of these methods show up in IntelliSense for me on this branch.

Reply:
Ah you're right. I thought those were moved out of the PR.

Reply:
They're actually supplied by `GenericCollectionAssertions` now.